graft.gemspec

```diff
@@ -9,11 +9,11 @@ Gem::Specification.new do |s|
   s.authors = ["Loic Nageleisen"]
   s.email = "loic.nageleisen@gmail.com"
   s.files = Dir.glob("lib/**/*.rb")
-  s.homepage = "https://rubygems.org/gems/graft"
+  s.homepage = "https://github.com/DataDog/graft-rb"
```

Comment:
Hopefully that fixes the homepage link. I seem to recall having seen some gems having something in `s.metadata` instead, maybe that's it?

Reply:
I think `homepage` and  metadata's `homepage_uri` are the same.